src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/StartGame.java

```diff
@@ -101,9 +262,287 @@ public StartGame(int connectionHashCode, UserMessage userMessage, Messenger mess
+            ArrayList<MinesCardDataResponse> secondLevelMinesCardsResponses=createMinesCardDataResponses(game.getRevealedCards(CardTier.LEVEL_2));
+            ArrayList<MinesCardDataResponse> thirdLevelMinesCardsResponses=createMinesCardDataResponses(game.getRevealedCards(CardTier.LEVEL_3));
+
+            UserDataResponse userToPlayResponse=new UserDataResponse(game.getCurrentPlayer().getUuid());
+                  
+            ResponseData responseData = new ResponseData(
+                tokensDataResponse, nobleDataResponses, 
+                firstLevelMinesCardsResponses, secondLevelMinesCardsResponses, 
+                thirdLevelMinesCardsResponses,userToPlayResponse
+            );
+
+
+            ArrayList<User> players= room.getAllUsers();
+            ServerMessage serverMessage = new ServerMessage(
+                    userMessage.getContextId(), 
+                    ServerMessageType.START_GAME_RESPONSE,
+                    Result.OK, 
+                    responseData
+            );
+            for(User player : players){          
+                messenger.addMessageToSend(player.getConnectionHashCode(), serverMessage);
+            }
+       
+
+        }catch (Exception e) {
+            ErrorResponse errorResponse = new ErrorResponse(Result.FAILURE, e.getMessage(), ServerMessageType.START_GAME_RESPONSE, userMessage.getContextId().toString());
+            messenger.addMessageToSend(connectionHashCode, errorResponse);
+        }
+    }
+
+
+
+
+
+
+
+
+    private void validateData(DataDTO dataDTO, Database database) throws  RoomDoesntExistException, UserDoesntExistException, RoomOwnershipException, InvalidUUIDException, RoomInGameException, RoomPlayerCountException{
+        Pattern uuidPattern = Pattern.compile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"); 
+
+         // Check if user UUID matches the pattern
+         Matcher uuidMatcher = uuidPattern.matcher(dataDTO.userDTO.uuid.toString());
+         if (!uuidMatcher.find())
+             throw new InvalidUUIDException("Invalid UUID format."); 
+ 
+         // Check if room UUID matches the pattern
+         uuidMatcher = uuidPattern.matcher(dataDTO.roomDTO.uuid.toString());
+         if (!uuidMatcher.find())
+             throw new InvalidUUIDException("Invalid UUID format."); 
+
+        User user = database.getUser(dataDTO.userDTO.uuid);
+        Room room =database.getRoom(dataDTO.roomDTO.uuid);
+
+        //Check if user exits
+        if (user==null){
+            throw new UserDoesntExistException("No such user in the database");
+        }
+
+        //Check if room exits
+        if(room==null){
+            throw new RoomDoesntExistException("No such room in the database");
+        }      
+
+        //Check if he owns only one room
+        int ownerships =0;
+        for(Room _room : database.getAllRooms()){
```

Comment:
Java variable naming convention. Useless function. User can only have one room. It is not possible to create more rooms, because of `CreateRoom Reaction` implementation.